internal/commands/inspect_builder.go

```diff
@@ -266,35 +272,152 @@ func runImagesOutput(runImage string, mirrors []string, cfg config.Config) (stri
 	return strings.TrimSuffix(buf.String(), "\n"), nil
 }
 
-func detectionOrderOutput(order dist.Order) (string, error) {
+type stackEntry struct {
+	LayerInfo dist.BuildpackRef
+	Depth     int
+	Last      bool
+}
+
+func reverseStack(stack []stackEntry) []stackEntry {
```

Comment:
nit: Can we move this method below where it's used, so readers understand the context?